[Docs] dashboard and ci docs (#1384)
The-OpenROAD-Project/OpenROAD-flow: docs/user/QoRDashboard.md

# Using the OpenROAD Dashboard to track and analyze Quality of Results

The metrics dashboard is tailored for RTL-to-GDS flow stages, 
targeting different designs across various platforms. The dashboard 

> comprises two main pages: the trends page and the comparison page. 

Reviewer: ```suggestion
# Using the OpenROAD Dashboard to Track and Analyze Quality of Results

The metrics dashboard (https://dashboard.theopenroadproject.org/)
is tailored for RTL-to-GDS flow stages, targeting different designs across
various platforms. The dashboard comprises two main pages: the trends
page and the comparison page. 
```

## Trends

The trends page allows users to visualize metric trends through 
interactive graphs, enabling the selection of platforms, designs, 
and metrics for each platform. This feature aids in tracking the 
fluctuations of metrics over time.

![Trends](../images/trends.webp)

## Comparison 

> The comparison page enables users to compare metrics between branches 
> or commits for a specific platform and its designs. The comparison is 
> shown in the form of a table that showcases the differences between the 
> base source and the test source. This page also offers deeper insights 
> like the platform summary table that presents statistical trends for 
> different designs within a platform. As well as, a summary table that 
> provides insights for the set of selected designs, with added 
> functionality for filtering. 

Reviewer: ```suggestion
The comparison page lets users compare metrics between branches 
or commits for a specific platform and its designs. The comparison is 
shown as a table showcasing the differences between the 
base source and the test source. This page also offers deeper insights, 
like the platform summary table that presents statistical trends for 
different designs within a platform and a summary table that 
provides insights for the set of selected designs, with added 
functionality for filtering. 
```

# Using the OpenROAD Dashboard to track and analyze Quality of Results

```{warning}
This page is under construction. Check back later!

> ```

Reviewer: @habibayassin
@luarss is working on our documentation, and we would like to include the great work you have done with the new dashboard. Could you help Jack write a small description of your work and create a user guide to showcase the currently deployed features?

Edit: I forgot to mention we can include a walkthrough like the above with Jenkins using a Google Slide presentation so we can keep images up to date easily.

Reviewer 2: @vvbandeira yes of course... @luarss I will send you an email with these details.
